### ISSUE TITLE
Fails to build with config on the latest hugo

### ISSUE THREAD

**Issue by DarrienG** [bbrks/hugo-gruvbox-theme#1]
Hi, not sure if you're still working on this anymore, but if you are this theme fails to build on the latest version of Hugo with the sample config. It fails with:

```
$ hugo
Start building sites …
hugo v0.93.2+extended darwin/arm64 BuildDate=unknown
ERROR 2022/03/05 18:37:24 render of "taxonomy" failed: "/Users/darrien/Developer/webdev/yeehaw/themes/gruvbox/layouts/_default/baseof.html:23:10": execute of template failed: template: _default/list.html:23:10: executing "_default/list.html" at <.Hugo.Generator>: can't evaluate field Hugo in type *hugolib.pageState
ERROR 2022/03/05 18:37:24 render of "taxonomy" failed: "/Users/darrien/Developer/webdev/yeehaw/themes/gruvbox/layouts/_default/baseof.html:23:10": execute of template failed: template: _default/list.html:23:10: executing "_default/list.html" at <.Hugo.Generator>: can't evaluate field Hugo in type *hugolib.pageState
Error: Error building site: failed to render pages: render of "home" failed: "/Users/darrien/Developer/webdev/yeehaw/themes/gruvbox/layouts/_default/baseof.html:23:10": execute of template failed: template: index.html:23:10: executing "index.html" at <.Hugo.Generator>: can't evaluate field Hugo in type *hugolib.pageState
Total in 16 ms
```

Hugo version:
```
🜛 blog [master●●] hugo version
hugo v0.93.2+extended darwin/arm64 BuildDate=unknown
```

And minimum reproducible example: https://gitlab.com/DarrienG/sample-broken-site

**Comment by bbrks** [comment]
Thanks for leaving an issue. I'll get around to this at some point to fix it.
I've neglected the theme for as long as I've neglected my blog!

**Comment by bbrks** [comment]
Should be fixed with 1356de3d6fafc66a6bfebd56c47e7064a13cf67b !

**Comment by DarrienG** [comment]
Thank you!
